js/client-rooms.js

```diff
@@ -209,10 +212,10 @@
 			return this.$list.html('<p>' + buf.length + (buf.length === 100 ? '+' : '') + ' ' + BattleLog.escapeFormat(this.format) + ' ' + (buf.length === 1 ? 'battle' : 'battles') + '</p>' + buf.join(""));
 		},
 		refresh: function () {
-			var elofilter = '';
-			var elo = this.$('select[name=elofilter]').val();
-			if (elo !== 'none') elofilter = ', ' + elo;
-			app.send('/cmd roomlist ' + this.format + elofilter);
+			var usernamePrefix = this.$('input[name=prefixsearch]').val();
+			var elofilter = this.$('select[name=elofilter]').val();
+			var searchParams = [this.format, elofilter, usernamePrefix];
```

Comment:
I don't think the space in `', '` is significant looking at the server side code.